adapters/repos/db/vector/hnsw/deserializer.go

```diff
@@ -143,6 +144,9 @@ func (d *Deserializer) Do(fd *bufio.Reader, initialState *DeserializationResult,
 		case AddPQ:
 			err = d.ReadPQ(fd, out)
 			readThisRound = 9
+		case AddSQ:
+			err = d.ReadSQ(fd, out)
+			readThisRound = 9
```

Comment:
@abdelr shouldn't this be 10 as `toWrite` is 11 for SQ?

Reply:
Fixed